projects/imagemagick/rotate_fuzzer.cc

```diff
@@ -0,0 +1,23 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  if (Size < sizeof(double)) {
+    return 0;
+  }
+  double Degrees = *reinterpret_cast<const double *>(Data);
+  if (!isfinite(Degrees)) {
+    return 0;
+  }
+  const Magick::Blob blob(Data + sizeof(double), Size - sizeof(double));
```

Comment:
using `sizeof(Degrees)` would be better, as it will automatically prevent you from getting into a trouble if type of `Degrees` will change to e.g. `float` or anything else